client/src/components/ExerciseCard.js

```diff
@@ -0,0 +1,26 @@
+import { useParams } from 'react-router';
+
+export default function ExerciseCard({ exercises }) {
+  const { id } = useParams();
+
+  const findExercise = exercises.find((exercise) => {
+    return exercise._id === id;
+  });
```

Comment:
A verb oriented names indicate this is a function, but it is the found object, maybe rename to `foundExercise`?